lambda/lambda_py/notifiers/format_utils.py

```diff
@@ -6,7 +6,10 @@
 def format_record(record: LogRecord) -> str:
     eventSource = record.raw_data.get("eventSource")
     eventName = record.raw_data.get("eventName")
-    return f"- At *{record.time}*, from *{record.sourceIPAddress}* - called {eventSource}:{eventName}"
+    return (
+        f"- At *{record.time}*, from *{record.sourceIPAddress}* "
+        f"- called {eventSource}:{eventName} via user agent {record.user_agent}"
```

Comment:
```suggestion
        f"- called {eventSource}:{eventName} via user agent '{record.user_agent}'"
```
Surrounding quotes will make it more explicit what exactly is part of the user agent.